packages/r/reproc/xmake.lua

```diff
@@ -0,0 +1,30 @@
+package("reproc")
+
+    set_description("a cross-platform C/C++ library that simplifies starting, stopping and communicating with external programs.")
+    set_homepage("https://github.com/DaanDeMeyer/reproc")
+    set_license("MIT")
+
+    add_urls("https://github.com/DaanDeMeyer/reproc.git")
+    add_versions("v14.2.4", "8aea3aef64c37a7ed0db759a7128a80c2540c808")
+
+    add_deps("cmake")
+
+    if is_plat("linux") or is_plat("mingw") then
+      add_syslinks("pthread")
```

Comment:
incorrect indent

Reply:
Oh shizzle, I'll wait for these checks to complete first and then push a quick fix.